docs/gifts.docs.js

```diff
@@ -0,0 +1,64 @@
+const giftsEndpoints = {
+  '/gifts/unpack': {
+    put: {
+      tags: ['gifts'],
+      summary: 'Put info',
+      description: 'gifts router',
+      operationId: 'giftsLoggedUser',
+      produces: ['application/json'],
+      security: {
+        api_key: {
```

Comment:
we don't need security option here or anywhere else, since we're passing directly Authorization hearer